.vscode/settings.json

```diff
@@ -15,5 +15,6 @@
     ],
     "cmake.buildDirectory": "${workspaceFolder}/build-vscode",
     "cmake.cmakePath": "${workspaceFolder}/.dependencies/cmake-3.15.5/bin/cmake",
-    "cmake.generator": "Ninja"
+    "cmake.generator": "Ninja",
```

Comment:
please don't include settings.json and similar preset configuration files in PR's